users/views.py

```diff
@@ -67,27 +67,26 @@ def login(request):
 
 @login_required
 def logout(request):
-    if request.user.is_authenticated:
-        django_logout(request)
-        response = '''
-                <p>Logout Successfully!! You will be redirected to login page in <span id="sp">1</span> seconds...</p>
-                <script>
-
-                    setInterval(go, 1000);
-                    var x=0;
-                    function go() {
-                        if (x>=0){
-                            document.getElementById("sp").innerText=x;
-                        } else {
-                            location.href="/user/login" ;
+    if request.method == 'POST':
+        if request.user.is_authenticated:
+            django_logout(request)
+            response = '''
+                    <p>Logout Successfully!! You will be redirected to login page in <span id="sp">1</span> seconds...</p>
```

Comment:
can you remove this javascript now and just redirect to the login page?

Reply:
Removed